### DISCUSSION TITLE
Visualisierungen für HA basierend auf open3e und E3onCAN

### DISCUSSION THREAD

**Discussion by MyHomeMyData** [MyHomeMyData/ha.vis.vitocal250#1] [Announcements]
## 👋 Willkommen!
  Hier ist der richtige Platz für 
  * Fragen, Ideen, Vorschläge
  * Vorstellen eigener Anpassungen und Erweiterungen
  * Gegenseitige Unterstützung

  Weiterführende Infos
  * [Viessmann-Forum](https://community.viessmann.de/t5/Konnektivitaet/CAN-Bus-Home-Automation-E3-Generation-lokal-und-kostenlos)
  * Projekt [open3e](https://github.com/open3e/open3e)
  * Projekt [E3onCAN](https://github.com/MyHomeMyData/E3onCAN)

  Eine entsprechende Visualisierung für [ioBroker](https://github.com/MyHomeMyData/iob.vis.vitocal250) ist ebenfalls verfügbar.

**Comment by aleex42** [comment]
Sehr geile Sache! 

Ich biete mich mal an, das auf eine Vitocal 200-S zu adaptieren. Kein großes Thema, aber halt anderes Hintergrundbild und bisschen andere Werte. Im Prinzip funktioniert es aber Out-of-the-Box mit deinen Werten/Konfigs, sieht nur komisch aus :-)

Einzige Frage die ich grade noch habe, wie kriegt man das Hintergrundbild so hin, dass es nicht komplett verpixelt den ganzen Bildschirm füllt.
Ich finde da leider keine Option um eine Breite einzustellen oder so.

**Reply by jwende** [reply]
Gerne ... ich habe 'nur' eine VC250 - also wäre das mein Fokus.
Mit dem Bild habe ich auch eine Weile experimentiert - außer panel: true ist mir auch nichts sinnvolles untergekommen - aber da skaliert sich das Image im jeweiligen Fenster. Und dann bei den picture-elements mit "vw" arbeiten - für die Optionen bei style auch immer die CSS Dokumentation zu Rate ziehen.
      - type: picture-elements
        elements:
          - type: image
            image: local/img/Heizwasserspeicheraus.png
            style:
              top: 40%
              left: 92%
              width: 3vw

**Reply by aleex42** [reply]
Das Bild hast du vom Bedienteil abfotografiert oder? :-)

**Reply by jwende** [reply]
Nein - aus dem GitHub vom ioBroker. Ich weiß nicht, wo Helmut es her hat.

**Reply by MyHomeMyData** [reply]
> Das Bild hast du vom Bedienteil abfotografiert oder? :-)
> 

Ja, vom Bedienteil zwei Versionen mit Stativ abfotografiert. In Photoshop die Zahlenwerte entfernt und beide Versionen (WP ein/aus) deckungsgleich zugeschnitten.

**Reply by aleex42** [reply]
Das werde ich wohl auch noch so machen müssen :-)

Soweit aber an die Vitocal 200-S angepasst - nur die "Ausgangstemperatur Außeneinheit" fehlt mir noch ...

![Screenshot 2024-03-20 183426](https://github.com/MyHomeMyData/ha.vis.vitocal250/assets/5489107/18eecb33-d120-4a3e-aef1-bebcca569b0c)

Details / Konfig kommt dann, wenn fertig :)

**Comment by Philip-Wiege** [comment]
Hallo zusammen!

Ich werde mich erst die nächsten Tage einbringen können. Bin aktuell familientechnisch etwas geerdet.

Langfristig plane ich noch zwei weitere Ansichten. Eine für die Inneneinheit und eine als Bedienpanel im Viessmann Stil für Heizung und WW.

Grüße
Philip

**Reply by jwende** [reply]
Perfekt - keine Hektik. Familie hat Vorrang.
Ich will jetzt als nächstes ein Element zum experimentieren einbauen - so dass man einen Request für eine DID absetzen kann und den Wert zurückbekommt.
Phase zwei wäre eine schreibende Schnittstelle für 'unverfängliche' Daten.
Gruß 
J.
